README.md

```diff
@@ -1,4 +1,6 @@
 # GitHub Gems: Driving Open-Source Investments With Data
+! [This is the lineage graph generated by dbt for the views in this repo.]
+(./docs/dbt_dag.png)
```

Comment:
It seems that the graph is not showing in the readme file. Maybe it's because there's a newline character between lines 2 and 3?